store/conf/attrs/zimbra-attrs.xml

```diff
@@ -1090,7 +1090,7 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
   <desc>whether IMAP is enabled for an account</desc>
 </attr>
 
-<attr id="175" name="zimbraPop3Enabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited,domainAdminModifiable">
+<attr id="175" name="zimbraPop3Enabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable">
```

Comment:
did you remove **accountInfo** as part of this work or it was before?

Reply:
I've removed accountInfo to maintain parity between IMAP and POP3 enable/disable mechanism.

Reply:
I've added the accountInfo back as i totally skip the backward compatibility part. Apologies.